prow/plugins/config.go

```diff
@@ -841,6 +851,28 @@ func (c *Configuration) ApproveFor(org, repo string) *Approve {
 	return a
 }
 
+// BlunderbussFor finds the BlunderbussConfig for an org or repo, if one exists.
+// Blunderbuss configuration can be listed for a repository or an organization.
+func (c *Configuration) BlunderbussFor(org, repo string) BlunderbussConfig {
```

Comment:
I would recommend name changing to be more descriptive of what the function does eg `FindExistingBlunderbussConfig`

Reply:
Sounds good to me! I was following naming of other functions, but I'm all about some more descriptive naming!

Reply:
That is my bad! I did not notice this name is standardized like this. It seems weird, but I guess it can stay this way.